react/index.tsx

```diff
@@ -1,17 +1,166 @@
 import { canUseDOM } from 'vtex.render-runtime'
-import { PixelMessage } from './typings/events'
+import { ProductOrder, Impression, PixelMessage } from './typings/events'
 
 export function handleEvents(e: PixelMessage) {
   switch (e.data.eventName) {
+    case 'vtex:userData': {
+      const { user } = e.data
+      epica('identify', user.id, {
+        isAuthenticated: user.isAuthenticated,
+        firstName: user.firstName,
+        lastName: user.lastName,
+        document: user.document,
+        id: user.id,
+        email: user.email,
+        phone: user.phone,        
```

Comment:
Do you really need to pass all this info? Is epica expecting in this format?

Reply:
FYI those fields might be undefined if the user is not authenticated or the user don't have input this data in their account yet.

Reply:
Ok, to identify the user in epica-pixel, the method receives as many fields as necessary. In this case I used them since the event provides them and their information is more complete.